providers/implementations/kdfs/tls1_prf.c

```diff
@@ -161,6 +165,7 @@ static void *kdf_tls1_prf_dup(void *vctx)
         if (!ossl_prov_memdup(src->seed, src->seedlen, &dest->seed,
                               &dest->seedlen))
             goto err;
+        OSSL_FIPS_IND_COPY(dest, src)
```

Comment:
Do you have a suggestion here?

ossl_FIPS_IND_callback() may be triggered from anywhere within some fips provider algorithm context - so it is not going to know anything about the cbarg or where to get it from per context. 
ossl_FIPS_IND_callback() calls the OSSL_INDICATOR_get_callback() in fipsprov.c which has to call some cb that has been passed from the core library (as well as finding the libctx of the core - since it has its own lib ctx).

Reply:
I suggest dropping the call back argument entirely.  Since there can only ever be one callback at any point in time and that the callback can be called from anywhere (thread, library, app, …), it doesn't make a lot of sense keeping it.

Reply:
since this is intended for logging I think removing it is probably the best thing to do here.

Reply:
removed cbarg